planet/src/main/FirebaseSecurityRules.txt

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains the security rules of the firebase realtime databse
+{
+    "rules": {
+        // Rules applied to the users tree
+        // Users can read other users' data and write their own data
+        "users": {
+            "$uid": {
```

Comment:
In order to get the profile using the username, we need to read the users tree. You haven't set any rules for this tree. I was wondering if it's possible to give access to the admin only? (like us developers)

Reply:
Actually, I think we can just leave it as it is but we'll have to create another tree for usernames that everybody can read from later when we add the privacy feature. For now, since all users can read the entire database data, I think we can just add after the 20th line: ".read": true,

Reply:
Okay!